tests/test_isrTaskLSST.py

```diff
@@ -400,6 +400,77 @@ def test_isrBias(self):
 
         self._check_bad_column_crosstalk_correction(result.exposure)
 
+    def test_isrBiasNoParallelOscanCorrection(self):
+        """Test processing of a bias frame with parallel
+        overscan correction turned off."""
+        mock_config = self.get_mock_config_no_signal()
+        mock_config.doAddParallelOverscanRamp = False
+
+        mock = isrMockLSST.IsrMockLSST(config=mock_config)
+        input_exp = mock.run()
+
+        isr_config = self.get_isr_config_electronic_corrections()
+
+        # Turn off the parallel overscan correction
+        ampConfig = isr_config.overscanCamera.getOverscanDetectorConfig(self.detector).defaultAmpConfig
```

Comment:
Small comment: this should be `amp_config` (snake_case) to match the style of the python file.